test/unit/bundle.test.js

```diff
@@ -0,0 +1,549 @@
+      .with.keys(['type', 'version']);
+    expect(result.output.data)
+      .to.have.length(1);
+    expect(
+      removeLineBreakTabsSpaces(result.output.data[0].rootFile.bundledContent)
+    ).to.equal(removeLineBreakTabsSpaces(expectedOutput));
+  });
+
+  it('Should bundle and dont remove the unresolved imports', async function() {
+    const serviceContent = fs.readFileSync(
+        path.join(__dirname, COUNTING_FOLDER, '/CountingCategoryService.wsdl'),
+        'utf-8'
+      ),
+      typesContent = fs.readFileSync(
+        path.join(__dirname, COUNTING_FOLDER, '/CountingCategoryData.xsd'),
+        'utf-8'
+      ),
+      expectedOutput = fs.readFileSync(
+        path.join(__dirname, COUNTING_FOLDER, '/output.wsdl'),
+        'utf-8'
+      ),
+      input = {
+        type: 'multiFile',
+        specificationVersion: '1.1',
+        rootFiles: [
+          {
+            path: '/CountingCategoryService.wsdl'
+          }
+        ],
+        options: {},
+        data: [
+          {
+            path: '/CountingCategoryData.wsdl',
+            content: typesContent
+          },
+          {
+            path: '/CountingCategoryService.wsdl',
+            content: serviceContent
+          }
+        ]
+      },
+      result = await Converter.bundle(input);
+    expect(result.result).to.be.true;
+    expect(result.output).to.be.an('object')
+      .with.keys(['data', 'type', 'specification']);
+    expect(result.output.specification).to.be.an('object')
+      .with.keys(['type', 'version']);
+    expect(result.output.data)
+      .to.have.length(1);
+    expect(
+      removeLineBreakTabsSpaces(result.output.data[0].rootFile.bundledContent)
+    ).to.equal(removeLineBreakTabsSpaces(expectedOutput));
+  });
+
+  it('Should bundle and remove resolved include tags', async function() {
+    const serviceContent = fs.readFileSync(
+        path.join(__dirname, INCLUDE_TAG, '/Services.wsdl'),
+        'utf-8'
+      ),
+      typesContent = fs.readFileSync(
+        path.join(__dirname, INCLUDE_TAG, '/Types.xsd'),
+        'utf-8'
+      ),
+      expectedOutput = fs.readFileSync(
+        path.join(__dirname, INCLUDE_TAG, '/output.wsdl'),
+        'utf-8'
+      ),
+      input = {
+        type: 'multiFile',
+        specificationVersion: '1.1',
+        rootFiles: [
+          {
+            path: '/Services.wsdl'
+          }
+        ],
+        options: {},
+        data: [
+          {
+            path: '/Types.xsd',
+            content: typesContent
+          },
+          {
+            path: '/Services.wsdl',
+            content: serviceContent
+          }
+        ]
+      },
+      result = await Converter.bundle(input);
+    expect(result.result).to.be.true;
+    expect(
+      removeLineBreakTabsSpaces(result.output.data[0].rootFile.bundledContent)
+    ).to.equal(removeLineBreakTabsSpaces(expectedOutput));
+  });
+
+  it('Should bundle and not remove missing imports', async function() {
```

Comment:
Q:What is the difference between this tests?

Should bundle and not remove missing imports
Should bundle and dont remove the unresolved imports